src/create-reminder-signup/lambda/lambda.ts

```diff
@@ -77,3 +77,16 @@ export const handler = async (
 		};
 	}
 };
+
+export const handler = (
+	event: APIGatewayEvent,
+	context: unknown,
+	callback: (err: Error | null, result?: APIGatewayProxyResult) => void,
```

Comment:
Looks like there might be a type in the sdk we could use:  `APIGatewayProxyCallback`

Reply:
Good spot!